### DISCUSSION TITLE
Ranking screen

### DISCUSSION THREAD

**Discussion by nelsonmestevao** [cesium/karaokium#7] [Ideas]
We should probably have a ranking screen that shows up on a new state called when the karaoke ends. At that moment we could have a confetti party and show all voting results averaged. Probably it would be cool to see all the time the top 3 performances, as well.

**Comment by GresosJ** [comment]
I think it should be possible to see the ranking screen during the karaoke, not just the top 3 performances. I think is a good dynamic seing the ranking changing during the karaoke.

**Reply by nelsonmestevao** [reply]
I guess it would be cool to have that screen in the TV and being the admin showing it.
